nautobot_device_lifecycle_mgmt/models.py

```diff
@@ -308,6 +447,9 @@ class InventoryItemSoftwareValidationResult(PrimaryModel):
     software = models.ForeignKey(
         to="dcim.SoftwareVersion", on_delete=models.CASCADE, help_text="Inventory Item software", blank=True, null=True
     )
+    old_software = models.ForeignKey(
+        to="SoftwareLCM", on_delete=models.CASCADE, help_text="Inventory Item software", blank=True, null=True
```

Comment:
Same as above, should this be:
```suggestion
        to="SoftwareLCM", on_delete=models.SET_NULL, help_text="Inventory Item software", blank=True, null=True
```

Reply:
Agreed. Let's set it to `models.SET_NULL`